### DISCUSSION TITLE
Tractor to stay on route?

### DISCUSSION THREAD

**Discussion by LindavK1972** [Courseplay/courseplay#7448] [Q&A] [answered]
Hi,

Is there a way to keep the tractor from taking shortcuts to like, a sell point or silo? I'm plaing on FS19 MVP and the map contains earth walls seperating fields. Evertime my tractor get's stuck on these walls cause it deviates from the course?

Thanks for the help in advance.

Kindest regards, Linda

**Comment by Tensuko** [comment]
No Idea what you talk about.
Sounds like Pathfinder can't find path and takes a direct way.
Or your Fieldcourse makes Problems in turn becouse you have not turned on Pathfinder in turns.
Screenshots ?

**Reply by STobi** [reply]
I have the same issue described by OP. I am using a telehandler to scoop from a hay pile into a trailer. I have defined a simple course for this. It works perfectly the first 3 loops but then afterwards, the telehandler will just ignore the course and instead drive straight into the back of the trailer to try and unload there where it fails and just gets stuck ramming into the back of the tailer. If I restart the driver then it will again work for 3 loops before doing the same behavior. I have tried several iterations of the course route but it happens each time.

**Reply by Tensuko** [reply]
More Informations, logfile, etc....

**Comment by LindavK1972** [comment]
![FarmingSimulator2019Game 2021-08-20 21-22-09](https://user-images.githubusercontent.com/15148234/130284091-f25e00c5-0e57-4504-9e58-bae449bd1fe5.png)

So, the route TO the silo, works great, but as you can see, the way back, cut's through all fields.

**Comment by LindavK1972** [comment, marked as answer]
But i worked around it with succes. Route to and from. So not one big route but split in half. To the silo and from the silo. Works like a charm!
